Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/meat.yml

```
@@ -54,6 +54,70 @@
     cycles: 1
     output: FoodBakedNugget
 
+- type: entity
+  name: raw fish fillet
+  parent: FoodMeatBase
+  id: FoodCuredCarp
+  description: A slice of treated alien space fish. It has been rendered safe to eat. Smells like space and fish.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - fishy
+  - type: Tag
+    tags:
+    - Raw
+    - Meat
+  - type: Sprite
+    sprite: _NF/Objects/Consumable/Food/meat.rsi
+    state: cured-carp
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        reagents:
+        - ReagentId: UncookedAnimalProteins
+          Quantity: 9
+        - ReagentId: Fat
+          Quantity: 9
+  
+- type: entity
+  name: battered raw fish fillet
+  parent: FoodCuredCarp
+  id: FoodCuredCarpBattered
+  description: A slice of treated alien space fish, dredged in a gloopy flour and egg mixture.
+  components:
+  - type: Sprite
+    sprite: _NF/Objects/Consumable/Food/meat.rsi
+    state: cured-carp-battered
+  - type: DeepFrySpawn
+    cycles: 1
```

```suggestion
    cycles: 2
```
Consistency with fried chicken, not nuggs.